projects/RabbitMQ.Client/client/logging/RabbitMqClientEventSource.cs

```diff
@@ -34,25 +34,20 @@
 
 namespace RabbitMQ.Client.Logging
 {
-    [EventSource(Name="rabbitmq-dotnet-client")]
-    public sealed class RabbitMqClientEventSource : EventSource
+    #nullable enable
+    internal sealed partial class RabbitMqClientEventSource : EventSource
     {
-        public class Keywords
+        public static readonly RabbitMqClientEventSource Log = new RabbitMqClientEventSource();
+
+        public RabbitMqClientEventSource()
+            : base("rabbitmq-dotnet-client")
```

Comment:
Do we want it to be named like that? or just "rabbitmq-client"

Reply:
This name would not conflict if another client one day provided the same metrics (e.g. Node?)

Reply:
I'd assume it's unlikely you'd use both in the same process, but I actually don't know what would happen if you do have 2 defining the same name. Let me follow up on this

Reply:
(unproven) looks like they would just be merged together probably.

Reply:
Then let's go with just "rabbitmq-client".

Reply:
fixed